content/en/blog/2022-10-04-prove-the-authenticity-of-the-oci-artifacts/index.md

```diff
@@ -0,0 +1,232 @@
+---
+author: developer-guy
+date: 2022-10-04 11:30:00+00:00
+title: Prove the Authenticity of the OCI Artifacts
+description: "We'll talk about integration of the cosign tool, which is a tool for signing and verifying the given container images, blobs, etc, that we used to prove the authenticity of the OCI Artifacts we manage through the OCIRepository resources."
+url: /blog/2022/10/prove-the-authenticity-of-the-oci-artifacts
+tags: [oci]
+resources:
+- src: "**.{png}"
+  title: "Image #:counter"
+---
+
+Software supply chain attacks are one of the most critical risks threatening today's software, and software has begun to collapse like a dark cloud over the industry. The Flux community is one of those communities that always takes precautions against these threats. While they primarily provided the measures I mentioned for their products, they also provided articles that raised users' awareness of what they were doing to protect against these threats. You can find all Flux's security articles [here](https://fluxcd.io/tags/security/). I strongly recommend you to read these articles. And they did not stop here, they also introduced features in their products that can protect users against these threats, and today we will talk about one of these features.
```

Comment:
Maybe explain what you mean by "software has begun to collapse like a dark cloud".

In this sentence there is also a bit of a mix of "I" and "we".

Reply:
There should be no "software" word in this sentence, and it's my mistake. I replace "I"s with "we"s.

Reply:
Have you done this already?

As you are a Flux Project Member, you are very free to write this blog post from the perspective of the Flux Project and say "we". Might make it easier to follow than reading a mix of "I", "we" and "they".